### PR TITLE
Readme rewrite

### PR DESCRIPTION
**Pull request by singular0**

### CODE REVIEW

**Review by SerGeRybakov — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,79 +1,7 @@
-# Deker tools
+# Deker Utility Functions
```
Comment (SerGeRybakov) [inline README.md]: I'm sure, that there is no reason to rename the project, repo and package.

File: README.md [hunk 2]
```diff
@@ -1,79 +1,7 @@
...
-This module provides ``get_utc`` function which returns timezone with UTC or current time by default:
-```python-repl
->>> get_utc()
-2023-07-26 15:42:05.539317+00:00
-```
-```python-repl
->>> from datetime import datetime
->>> get_utc(datetime.now())
-2023-07-26 15:42:05.539317+00:00
-```
```
Comment (SerGeRybakov) [inline README.md]: What was the reason to remove the examples? This package is independent from any other Deker deps and can be used in any project, not related to Deker. Any developer, who managed to reach this readme, could have a fast look over the examples and understand how to use this or that function. Without examples the repo became muted.

**Review by SerGeRybakov — CHANGES_REQUESTED** [review]
To my mind any readme **must help** undestanding the contents and aims of the project.
I think, we shall discuss this fix.

File: README.md (hunk 1 above)
Comment (matveyvarg) [reply README.md]: +1

File: README.md (hunk 2 above)
Comment (matveyvarg) [reply README.md]: IMHO, it's not a big deal as we have link to the documentation. However, I would agree with @SerGeRybakov. I believe that it's better to keep at least some examples here.

Comment (singular0) [reply README.md]: According to Deker documentation API of this package is not stable, so I this contradicts an idea to spin it off into separate self-sufficient project

**Review by SerGeRybakov — APPROVED** [review]

**Pull request merged**